integration_tests/src/lib.rs

```diff
@@ -5695,13 +5695,13 @@ mod test {
             blocks_per_extent: sv_tds.blocks_per_extent(),
             extent_count: sv_tds.extent_count(),
             opts: sv_opts.clone(),
-            gen: 1,
+            gen: 2,
```

Comment:
Why does this `gen` need to change?  I would expect to only need a new `gen` for the replacement (below)

Reply:
This one does not.  I initially changed the wrong generation spot.  I've reverted this change.